test/OgmiosDatumCache.purs

```diff
@@ -27,31 +23,14 @@ import Test.Spec.Assertions (shouldEqual)
 
 suite :: TestPlanM (Aff Unit) Unit
 suite = group "Ogmios Datum Cache tests" $ do
+  -- TODO Move
   skip $ test
     "Plutus data samples should satisfy the Aeson roundtrip test (FIXME: \
     \https://github.com/mlabs-haskell/purescript-aeson/issues/7)"
     plutusDataToFromAesonTest
   test "Plutus data samples should have a compatible hash" plutusDataHashingTest
-  test "GetDatumsByHashesR fixture parses and hashes properly"
-    getDatumsByHashesHashingTest
 
-readGetDatumsByHashesSample
-  :: forall (m :: Type -> Type)
-   . MonadEffect m
-  => m GetDatumsByHashesR
-readGetDatumsByHashesSample = do
-  errEither <<< decodeAeson =<< readAeson
-    "./fixtures/test/ogmios-datum-cache/get-datums-by-hashes-samples.json"
-
-getDatumsByHashesHashingTest
-  :: forall (m :: Type -> Type)
-   . MonadEffect m
-  => MonadThrow Error m
-  => m Unit
-getDatumsByHashesHashingTest = do
-  datums <- Map.toUnfoldable <<< unwrap <$> readGetDatumsByHashesSample
-  for_ (datums :: Array _) \(hash /\ datum) -> do
-    (datumHash <$> datum) `shouldEqual` Right (Just hash)
+-- TODO Add GetTxByHash
```

Comment:
Is TODO resolved?

Reply:
Not quite the same, the AffInterface only checks for any response from the server, not a parse of the success case. That being said, we only used it for transaction confirmation, and now we don't use it at all, due to Kupo replacing it. We should strip out all ODC stuff imo